extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -94,17 +101,29 @@ export function VideoWrapper({
   const readyDeferred = useMemo(() => new Deferred(), []);
 
   const play = useCallback(() => {
-    readyDeferred.promise.then(() => {
-      playerRef.current.play();
-    });
+    return readyDeferred.promise.then(() =>
+      Promise.resolve(playerRef.current.play()).catch(() => {
+        // Empty catch to prevent useless unhandled rejection logging.
```

Comment:
Hmm. Didn't we decide to propagate this error and let AMP layer ignore it instead?

Reply:
Moved to autoplay call-site as discussed [previously.](https://github.com/ampproject/amphtml/pull/30280#discussion_r494486274)